### ISSUE TITLE
Advance install scripts to Django 3.0

### ISSUE THREAD

**Issue by davidpablocohn** [OceanDataTools/openrvdas#212]
Django V2 has some bugs they're not going to fix. V3 appears stable and relatively mature by now.

**Comment by davidpablocohn** [comment]
The release notes for 3.0 flag incompatibilities between 2 and 3: https://docs.djangoproject.com/en/3.0/releases/3.0/#backwards-incompatible-3-0

The main salient one is that when Model.save() is called with a primary key, it now does an insert of a new entry rather than looking for and updating the entry with the specified key. The doc says:

_In order to update an existing model for a specific primary key value, use the update_or_create() method or QuerySet.filter(pk=…).update(…) instead. For example:_
```
>>> MyModel.objects.update_or_create(pk=existing_pk, defaults={'name': 'new name'})
>>> MyModel.objects.filter(pk=existing_pk).update(name='new name')
```

**Comment by davidpablocohn** [comment]
The thing that's really preventing this is that Django 3.0 requires Sqlite  > 3.8. CentOS uses an old version, and I can't figure out how to install the newer one and have Python find it.

**Comment by davidpablocohn** [comment]
Fixed with v1.1.0
